packages/client/src/products/types/productVariant.types.ts

```diff
@@ -1,6 +1,14 @@
 import type { Attribute, PurchaseChannel } from '../../types/index.js';
 import type { Price } from './price.types.js';
 
+type PublicationFeature = {
+  /**
+   * Code of the publication feature.
+   * For example: "IMMEDIATE_FULFILMENT", "PREORDER", "BACKORDER".
+   */
+  code: string;
```

Comment:
We know the ones provided in the example "IMMEDIATE_FULFILMENT", "PREORDER", "BACKORDER" but the service only indicates it's a string. How do you suggest we do this?

Reply:
You can provide an enum with the codes you know like this:

```js
export enum PublicationFeatureCode {
  ImmediateFulfilment: 'IMMEDIATE_FULFILMENT',
  PreOrder: 'PREORDER',
  BackOrder: 'BACKORDER'
}
```

That way the consumer can use them to compare (I assume the consumer might need to do it. If it might not be necessary, then we can discard the enum altogether)

Reply:
Done. Can you please revalidate?

Reply:
It is ok, but you will need to update the index test as it will be added to the snapshot.